src/routes/categories/components/CategoryEntry.js

```diff
@@ -0,0 +1,175 @@
+// @flow
+
+import * as React from 'react'
+
+import { CategoryModel } from '@integreat-app/integreat-api-client'
+import iconPlaceholder from '../assets/IconPlaceholder.svg'
+import styled, { withTheme } from 'styled-components'
+import Highlighter from 'react-highlight-words'
+import normalizeSearchString from '../../../modules/common/utils/normalizeSearchString'
+import Link from 'redux-first-router-link'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+import { fromString as htmlToText } from 'html-to-text'
+
+const NUM_WORDS_SURROUNDING_MATCH = 3
```

Comment:
Not sure if we should increase this number. Not sure though. Maybe put it up to a vote in #app.

Reply:
I guess we agreed on sth like 8+. Therefore 10 might be a good number